geode-core/src/main/java/org/apache/geode/examples/security/ExampleSecurityManager.java

```diff
@@ -257,15 +257,65 @@ private void readUsers(final Map<String, User> rolesToUsers, final JsonNode node
     return roleMap;
   }
 
-  static class Role {
+  public static class Role {
     List<ResourcePermission> permissions = new ArrayList<>();
+
```

Comment:
I meant those getters and setters inside class User and class Role. Those were not there before your change set unless I am missing something.....